javascript/packages/orchestrator/src/configGenerator.ts

```diff
@@ -529,11 +529,20 @@ async function getCollatorNodeFromConfig(
   const ports = await getPorts(provider, collatorConfig);
   const externalPorts = await getExternalPorts(provider, ports, collatorConfig);
 
+  // By default cumulus collators are `validators`, this implies that we will
+  // run those with this flag `--collator`.
+  const isValidator =
+    collatorConfig.validator !== undefined
+      ? collatorConfig.validator
+      : cumulusBased
+      ? true
+      : false;
```

Comment:
I'm having a hard time parsing this nested ternary statement. Also, I think JS has a `??` operator or something similar you could use, IIRC.

Reply:
Correct. I think this could simplier be:
```suggestion
  const isValidator = !!collatorConfig.validator || !!cumulusBased;
```

Reply:
Sounds good to me, old ternary addiction 😂

we can also use 
```
const isValidator = Boolean(collatorConfig.validator) || Boolean(cumulusBased);
```

Thanks!!

Reply:
I just add a better comment about this ternary, resolving here.
Thx!